pkg/statsd/statsd_defaults_and_params.go

```diff
@@ -180,6 +184,7 @@ func AddFlags(fs *pflag.FlagSet) {
 	fs.Bool(ParamConnPerReader, DefaultConnPerReader, "Create a separate connection per reader (requires system support for reusing addresses)")
 	fs.String(ParamServerMode, DefaultServerMode, "The server mode to run in")
 	fs.String(ParamHostname, getHost(), "overrides the hostname of the server")
+	fs.Bool(ParamLogRawMetric, DefineLogRawMetric, "Print metrics received from network to stdout in JSON format")
```

Comment:
It just logs it - whether it's to stdout or is JSON formatted is dependent on other configuration.